### PR TITLE
Refactored WireHarnesses

### LINKED ISSUE #41: Refactor Wire Harnesses Component
image

Create a JSON array in the `data` folder called `WireHarnesses.js`

Map through this array in the `WireHarnesses.jsx` component.

This will significantly clean up code.

### PR DESCRIPTION
**Pull request by Kolzza**
image

I cant figure out how to line up the margins of each prop. Idk if this is a problem though cause it was like this for the first pull request too but it was accepted.

### CODE REVIEW

File: src/data/HarnessAssembly.js [hunk 1]
```diff
@@ -1,7 +1,27 @@
+import WireHarness1 from "@/public/images/wireharness/WireHarness1.webp";
```
Comment (SafeDuck) [inline src/data/HarnessAssembly.js:1]: You edited a different file. Check the issue, we would like you to create a new file

File: src/components/harnesses/WireHarnesses.jsx [hunk 2]
```diff
@@ -1,39 +1,19 @@
...
-        img={WireHarness3}
-      />
-      <Description
-        text={
-          "Offering flexible solutions tailored to the customer, we have the technologies, expertise, and infrastructure to carry out build-to-print projects with the efficiency and responsiveness that customers expect. Out manufacturing engineers are involved as needed throughout the process to facilitate manufacturability, maximize cost-savings through material recommendations, and minimize waste."
-        }
-        img={WireHarness4}
-        float={"right"}
-      />
+    <div className="mx-40 flex items-center flex-col gap-10 px-3">
```
Comment (SafeDuck) [inline src/components/harnesses/WireHarnesses.jsx]: mx-40 is too much margin, it wont scale well

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/data/WireHarness.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    img: WireHarness3,
+  },
+  {
+    text: "Offering flexible solutions tailored to the customer, we have the technologies, expertise, and infrastructure to carry out build-to-print projects with the efficiency and responsiveness that customers expect. Out manufacturing engineers are involved as needed throughout the process to facilitate manufacturability, maximize cost-savings through material recommendations, and minimize waste.",
+    img: WireHarness4,
+    float: "right",
+  },
+];
+
+// Add the rest of the bullet points here. Delete this comment once you push
```
Comment (SafeDuck) [inline src/data/WireHarness.js:27]: :sob:

Comment (Kolzza) [inline src/data/WireHarness.js]: 💀

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**